docs/tutorial/linux-desktop-actions.md

```diff
@@ -1,7 +1,9 @@
-# Unity Launcher Shortcuts
+# Custom Linux Desktop Launcher Actions
 
-In Unity, you can add custom entries to its launcher via modifying the
-`.desktop` file, see [Adding Shortcuts to a Launcher][unity-launcher].
+On many Linux environments, you can add custom entries to its launcher
+by modifying the `.desktop` file. For Canonical's Unity documentation,
+see [Adding Shortcuts to a Launcher][unity-launcher]. For details on a
+more generic implementation, see the [Free Desktop Specification][spec].
```

Comment:
I would say `freedesktop.org specification` instead of `Free Desktop Specification`.